io.github.teamclouday.android-mic.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "io.github.teamclouday.android-mic",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "android-mic",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro",
+    "--device=usb",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-run/pipewire-0 "
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "env": {
+      "CARGO_HOME": "/run/build/android-mic/cargo"
+    }
+  },
+  "modules": [
+    {
+      "name": "protoc",
+      "buildsystem": "simple",
+      "build-commands": ["install -Dm0755 bin/protoc /app/bin/protoc"],
+      "cleanup": ["*"],
+      "sources": [
+        {
+          "type": "archive",
+          "strip-components": 0,
+          "dest-filename": "protoc.zip",
+          "url": "https://github.com/protocolbuffers/protobuf/releases/download/v29.3/protoc-29.3-linux-x86_64.zip",
```

Comment:
why don't you build from source?

Reply:
it's simpler this way i think

Reply:
no it's not.

Reply:
why?

Reply:
I think not building this from source is OK, as it is being cleaned up later. But also I don't think it requires a lot of dependencies, I see a bunch of apps building it without much. https://github.com/flathub/re.sonny.Workbench/blob/cd9237afb29ea9c6d5f356ba3a50751c294ab707/modules/libshumate.json#L14-L25